cbw_api_toolbox/cbw_api.py

```diff
@@ -66,11 +66,12 @@ def _cbw_parser(self, response):
         """Parse the response text of an API request"""
         try:
             result = json.loads(response.text, object_hook=lambda d: namedtuple('cbw_object', d.keys())(*d.values()))
+            return result
         except TypeError:
             self.logger.error("An error occurred while parsing response")
-        return result
+            sys.exit(-1)
```

Comment:
I'm not sure about this modification. The motivation was to avoid having `None` as a return value for this. But this is not related at all to this PR and exiting here might not be the best behavior.